### DISCUSSION TITLE
Unsharp text rendering

### DISCUSSION THREAD

**Discussion by simonderks** [pointfreeco/swift-snapshot-testing#672] [General]
Hi all,

I noticed unsharp text element rendering in snapshots using SwiftUI views, resulting in failing snapshot tests.
The issue occurred to me for the first time after updating to macOS Ventura & Xcode 14.1
Did anyone notice the same?

**Comment by teejayhh** [comment]
I have seen something similar for Xcode14.1  it happens on only one screen but on a few sims and I can't explain why. 

The red is the diff, which shows that letters moved slightly. There where no font adjustments and and no code changed. It worked the day before and now its being a little annoying. 

<img width="1508" alt="Screen Shot 2022-11-21 at 9 20 49 am" src="https://user-images.githubusercontent.com/2525235/202929410-ca58964d-012b-4e1e-9101-d806d0a175aa.png">

**Comment by PierreCapo** [comment]
We also have the exact same problem with Xcode 14.0 and a custom font. @teejayhh @simonderks  are you using a custom font also or does this happen with the default system san francisco font ?

<img width="155" alt="image" src="https://user-images.githubusercontent.com/26744253/207548102-4edf426b-9837-4297-8463-ab9767aae83c.png">

**Reply by simonderks** [reply]
No, we are only using system fonts. the issue occurs using the default font.

**Reply by teejayhh** [reply]
Yeah we are also only using the system font.
